client/components/content-containers/tcc-exam/edit/AssessmentGroup.vue

```diff
@@ -0,0 +1,195 @@
+<template>
+  <div class="assessment-group">
+    <div :class="{ divider: position }"></div>
+    <span @click="$emit('delete')" class="remove">
+      <span class="mdi mdi-delete"></span>
+    </span>
+    <div class="form-inline pull-right">
+      <div class="form-group time-limit">
+        <label for="timeLimit">Time limit (minutes)</label>
+        <input
+          v-model="timeLimit"
+          id="timeLimit"
+          class="form-control"
+          type="number"
+          step="15">
+      </div>
+    </div>
+    <h3>Question group {{ position | toLetter }}</h3>
+    <h4>Introduction</h4>
+    <group-introduction
+      @saveElement="$emit('saveElement', $event)"
+      @reorderElement="$emit('reorderElement', $event)"
+      @deleteElement="$emit('deleteElement', $event)"
+      :group="group"
+      :tes="tes" />
+    <h4>Questions</h4>
+    <div v-if="!hasAssessments" class="well">
+      Click the button below to Create first Assessment.
+    </div>
+    <element-list
+      @add="addAssessment"
+      @reorder="$emit('reorderElement', $event)"
+      :elements="assessments"
+      :activity="group"
+      :supported-types="['ASSESSMENT']">
+      <template v-slot:list-item="{ element }">
+        <assessment-item
+          @save="saveAssessment"
+          @delete="deleteElement(element)"
+          :assessment="element"
+          :objectives="objectives"
+          :objective-label="objectiveLabel" />
+      </template>
+    </element-list>
+  </div>
+</template>
+
+<script>
+import AssessmentItem from './Assessment';
+import cloneDeep from 'lodash/cloneDeep';
+import cuid from 'cuid';
+import debounce from 'lodash/debounce';
+import { ElementList } from 'tce-core';
+import filter from 'lodash/filter';
+import get from 'lodash/get';
+import { getLevel } from 'shared/activities';
```

Comment:
This can be removed if we just always show the `"Objective"` label or add a label option to the config which defaults to `"Objective"`. It was this way to support multiple use-cases but only AICPA uses Exams and only has one Objective type.